include/edhoc_rs.h

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
I don't understand why is this file included in the repo when it seems to be generated on the fly by the `build_for_c.sh` script?

Reply:
Yes, I just wanted to put the header files _somewhere_. Maybe:
- move the generated include folder to `./target/include` and
- add to the releases as a generated artefact, just like the fstar files?

Reply:
Moved to `target/include`.